### PR TITLE
CompiledCode-useLiteralAt

### PR DESCRIPTION
**Pull request by MarcusDenker**
antother fix..

### CODE REVIEW

**Comment by MarcusDenker** [comment]
This fixes e.g.

(SendsDeprecatedMethodToGlobalRuleTest>>#testBasicCheck ) hasLiteral: (SendsDeprecatedMethodToGlobalRule binding).

File: src/Kernel/CompiledCode.class.st [hunk 1]
```diff
@@ -235,11 +235,11 @@ CompiledCode >> frameSize [
 { #category : #literals }
 CompiledCode >> hasLiteral: literal [
 	"Answer whether the receiver references the argument, literal."
-	2 to: self numLiterals - 1 do: "exclude superclass + selector/properties"
+	1 to: self numLiterals - self literalsToSkip do: "exclude superclass + selector/properties"
```
Comment (dionisiydk) [inline src/Kernel/CompiledCode.class.st:238]: Now it starts with 1. Do you know why it was 2 before?

Comment (MarcusDenker) [reply src/Kernel/CompiledCode.class.st:238]: yes, is was 2 to - 1, now it is 1 to -2, but #literalAt: does "+1".

**Comment by Ducasse** [comment]
This looks good to me. This is nice to see this code improving.

**Pull request merged**